src/parse/td1.ts

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
 import { formats } from '../formats';
-
-import { getResult } from './getResult';
-import { ParseMRZOptions } from './parse';
+import { ParseMRZOptions } from '../types';
+import getResult from './getResult';
 import TD1Fields from './td1Fields';
 
 const TD1 = formats.TD1;
+/**
+ * It takes an array of three strings, each of which is 30 characters long, and returns an object with
+ * the parsed data
+ * @param {string[]} lines - An array of strings, each string representing a line of the MRZ.
+ * @param {ParseMRZOptions} options - ParseMRZOptions
+ * @returns the result of the getResult function.
+ */
 export default function parseTD1(lines: string[], options: ParseMRZOptions) {
```

Comment:
Could you add types in return values to all the parsers (this one but also TD2, TD3 etc...)